client/store/redux/users/usersReducer.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * ACTION TYPES
+ */
+const SET_USERS = 'SET_USERS'
+const SET_A_USER = 'SET_A_USER'
+const DELETE_USER = 'DELETE_USER'
+
+/**
+ * INITIAL STATE
+ */
+// const defaultUser = {}
+// in JPFP started with below. users.js in BoilerMaker is above
+const initialState = []
+
+/**
+ * ACTION CREATORS
+ */
+export const destroyUser = user => ({
+  type: DELETE_USER,
+  user: user
+})
+export const setUser = user => ({
+  type: SET_A_USER,
+  user: user
+})
+export const setUsers = users => ({
+  type: SET_USERS,
+  users: users
+})
+
+/**
+ * THUNK CREATORS
+ */
+
+export const deleteUser = user => async (dispatch, getState, {axios}) => {
```

Comment:
Potentially remove "getState, {axios}" from thunk creators

Reply:
imported

Reply:
see new note in this file